spec/features/rag_wise/forum_topic_spec.rb

```diff
@@ -198,7 +198,7 @@
         find('#select').click
         find('#select-question').click
 
-        expect_any_instance_of(Course::Forum::TopicsController).to receive(:auto_answer_action).with(false).once
+        expect_any_instance_of(Course::Forum::TopicsController).to receive(:auto_answer_action).once
 
         find('button.btn-submit').click
         wait_for_page
```

Comment:
If not needed, then remove these.  
Same for L163-L164

Reply:
Removing wait_for_page makes the test cases flaky as it might not receive the signal for auto_answer yet